webxrlayers-1.bs

```diff
@@ -500,6 +500,10 @@ of either a {{WebGLRenderingContext}} or a {{WebGL2RenderingContext}}.
 Each {{XRWebGLLayerFactory}} has an associated <dfn for="XRWebGLLayerFactory">session</dfn>, which is the 
 {{XRSession}} it was created with.
 
+NOTE: it is possible to create more than one {{XRWebGLLayerFactory}}. Any layer created with an instance of {{XRWebGLLayerFactory}} can be used
+ with layers created with other instances of {{XRWebGLLayerFactory}} as long as they were created with the same [=XRWebGLLayer/context=] and 
```

Comment:
> as long as they were created with the same `context`

This somewhat implies that we wouldn't be able to mix WebGL layers and video layers, which I'm quite certain isn't what any of us actually want. What's the intent behind this restriction? I think it just needs to be worded differently.

Reply:
yes, my bad. `context` needs to be removed.

Reply:
@toji I ended up rewording both sections.
Let me know what you think.